src/Spice86.Core/Emulator/Devices/Video/AeonCard.cs

```diff
@@ -254,10 +261,10 @@ public class AeonCard : DefaultIOPortHandler, IVideoCard, IAeonVgaCard, IDisposa
                 if (previousVerticalEnd != CrtController.VerticalDisplayEnd) {
                     ChangeVerticalEnd();
                 }
-
                 break;
             default:
-                throw new InvalidOperationException($"Writing 0x{value:X2} to port 0x{port:X4} is not (yet) supported by this VGA controller.");
+                base.WriteByte(port, value);
```

Comment:
I'm not sure. 
There are 2 cases in which we hit this situation:
1. the `AeonCard.WriteByte` method is being called with a port it did not register itself for.
2. the `AeonCard.WriteByte` method is being called with a port it can only read from.

In case 1 I think we should still have an exception. For case 2 calling the base class seems fine.

This also loses logging of the value being written. Maybe that should be included in the base class?

Reply:
For case 1, the base class handles this. If there is no-one registered, and -f was passed on the command line, an exception is thrown.
For case 2, the base class handles this, if -f was passed on the command line, and the child class called the base class.

Reply:
Case 1 is when the Aeon class did _not_ register the port, but the **method** in the Aeon class is called with that port anyway. 
Are we sure we want to ignore this if the -f flag is not set?

Reply:
Ah, I see. Called by overrides. Good point.

I think this somehow should be handled by the Base Class for every case, not just the Aeon Card. Maybe we'll have to call an inherited guard method in all child class... Doesn't feel like the cleanest solution.

Reply:
I think the most likely cause would be the Word and DWord versions of the port accessing API.
We can fix that by changing the `IOPortDispatcher` to get he port for each byte and only call WriteByte.

Not something to fix in this PR probably.